lib/FirebaseUtilities.ts

```diff
@@ -137,18 +137,25 @@ export async function dislikeLand(
   address: string,
   metaverse: Metaverse
 ) {
-  const land = doc(db, 'lands-' + metaverse, landId)
-  const landData = (await getDoc(land)).data() as Score | undefined
+  const land = doc(db, "lands-" + metaverse, landId);
+  const landData = (await getDoc(land)).data() as Score | undefined;
   // If user already disliked then take that dislike away
   if (landData?.dislikes.includes(address))
     return await updateDoc(land, {
       dislikes: arrayRemove(address),
-    })
+    });
 
   await updateDoc(land, {
     dislikes: arrayUnion(address),
-  })
+  });
   await updateDoc(land, {
     likes: arrayRemove(address),
-  })
+  });
+}
+
+// Retrieve valuation daily data
+export async function getValuationDailyData(metaverse: Metaverse) {
+  var valuesDoc = doc(db, "valuationGraphs", metaverse);
```

Comment:
This I think I told you on slack. But try to use `const` and `let` instead. Its a bit more type safe and helps other developers understand your code faster and not change things that shouldn't be changed.